HelpAssistHer/help-assist-her
# Add tabs component

## Description
This PR adds the tabs for the Pregnancy Resource Center form and the Community Health Center form.
Designs: https://projects.invisionapp.com/d/main#/console/11323423/296048040/preview

## Test Plan
1. Load the app
1. Verify you see the 2 tabs at the top and that they match the designs
1. Click on the Community Health Center tab and verify it loads the CommunityHealthCenter component (just has text on the page)
1. Click on the Pregnancy Resource Center tab and verify it loads the PRC form

## Regression Test Plan
1. Click Verify Next Resource and make sure it loads a resource
1. Click Save and verify that data can still be saved successfully
